cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SpatialFunctionsTest.scala

```diff
@@ -48,6 +48,14 @@ class SpatialFunctionsTest extends DocumentingTest {
     override def getCRS: spatial.CRS = crs
 
     override def getCoordinates: util.List[Coordinate] = List(new Coordinate(x, y)).asJava
+
+    override def equals(obj: Any): Boolean = {
+      obj match {
+        case other: Point =>
+          (other.getCRS.getHref == this.getCRS.getHref) && other.getCoordinates == this.getCoordinates
+        case _ => false
+      }
+    }
```

Comment:
This equals was not symmetric before, and the way the collections are compared in 2.13 seems to have changed.